shenyu-dist/shenyu-admin-dist/docker/Dockerfile

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM alpine AS prepare
+FROM centos:7 AS prepare
```

Comment:
alpine is 5.9M, but centos 7 is 200M, do you compare the package size?

Reply:
i will choose another base image later

Reply:
@moremind  hi, i have replace the base image. refer to https://thecattlecrew.net/2022/11/07/preparing-for-spring-boot-3-choose-the-right-java-17-base-image/